README.md

```diff
@@ -148,6 +148,7 @@ For local development and familiarizing yourself with Storm/Mesos, please see th
 * `mesos.master.failover.timeout.secs`: Framework failover timeout in second. Defaults to "24*7*3600".
 * `mesos.allowed.hosts`: Allowed hosts to run topology, which takes hostname list as a white list.
 * `mesos.disallowed.hosts`: Disallowed hosts to run topology, which takes hostname list as a back list.
+* `mesos.constraints`: Constrain slave nodes which bootstrap supervisor based on mesos-attributes.  
```

Comment:
Some typical examples are cited as follows.
- Apache spark
   https://github.com/apache/spark/blob/master/resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala#L270-L294
- Kafka Mesos Framework
   https://github.com/mesos/kafka/blob/master/src/scala/main/ly/stealth/mesos/kafka/Broker.scala#L80-L84
   https://github.com/elodina/java-mesos-util/blob/master/src/java/net/elodina/mesos/util/Constraint.java
- Marathon Framework
   https://mesosphere.github.io/marathon/docs/constraints.html#cluster-operator
   
I mimicked `CLUSTER operator` of Marathon Framework in these framework.